scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -35,13 +35,11 @@ workflow GvsCreateFilterSet {
   # fully-qualified table names
   String fq_sample_table = "~{project_id}.~{dataset_name}.sample_info"
   String fq_alt_allele_table = "~{project_id}.~{dataset_name}.alt_allele"
-  String fq_info_destination_table = "~{project_id}.~{dataset_name}.filter_set_info"
-  String fq_info_destination_table_vqsr_lite = "~{project_id}.~{dataset_name}.filter_set_info_vqsr_lite"
-  String fq_tranches_destination_table = "~{project_id}.~{dataset_name}.filter_set_tranches"
-  String fq_filter_sites_destination_table = "~{project_id}.~{dataset_name}.filter_set_sites"
+  String fq_filter_sites_destination_table =    "~{project_id}.~{dataset_name}.filter_set_sites"
```

Comment:
not in the scope of this pr necessarily, but why do these need to be fully qualified with the project id?

Reply:
Hmm. Really not sure. I was just following the pattern used already. I suppose it's good form?